src/parser/parser.ee.ts

```diff
@@ -51,6 +67,21 @@ async function getMimeType({ documentName }: { documentName: string }) {
   if (documentName.endsWith('.docx')) {
     return 'application/vnd.openxmlformats-officedocument.wordprocessingml.document';
   }
+  if (documentName.endsWith('.pptx')) {
+    return 'application/vnd.openxmlformats-officedocument.presentationml.presentation';
+  }
+  if (documentName.endsWith('xlsx')) {
+    return 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet';
+  }
```

Comment:
tiny details, but sort them the same way you would sort the supported file check

it makes it easier to maintain when they have the same sort (especially when we will have tons of file types)